index.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
+/* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(function (fn) { /* eslint prefer-arrow-callback: 1*/
```

Comment:
Eventually we will use es6 for everything (when we have time to change `conventional-changelog`).